private/sass/layout/_iconography.scss

```diff
@@ -24,1633 +23,1633 @@
 // INFO: use unicode characters for accessibility reasons and use aria-hidden="true" for decorative icons
 // DOCS: http://filamentgroup.com/lab/bulletproof_icon_fonts.html
 .fa-glass:before {
-  content: "\f000";
+    content: "\f000";
```

Comment:
I agree, yet it comes in hand as we have to use the correct formatting. In a global range, reformatting code should be added within a separate PR.